go/vm/geth/ct.go

```diff
@@ -202,7 +202,27 @@ func (i *callInterceptor) CallCode(env *geth_vm.EVM, me geth_vm.ContractRef, add
 }
 
 func (i *callInterceptor) DelegateCall(env *geth_vm.EVM, me geth_vm.ContractRef, addr geth_common.Address, data []byte, gas uint64) ([]byte, uint64, error) {
-	panic("not implemented")
+	kind := vm.DelegateCall
+	if i.static {
+		kind = vm.StaticCall
+	}
```

Comment:
Is this true? Why would DelegateCalls in static contexts be replaced by Static calls?